wire-ios-sync-engine/Tests/Source/Integration/IntegrationTest+Encryption.swift

```diff
@@ -126,7 +126,7 @@ extension IntegrationTest {
             return (localClient, lastPrekey)
         }
 
-        let selfClient = await context.perform { ZMUser.selfUser(in: context).selfClient()! }
+        _ = await context.perform { ZMUser.selfUser(in: context).selfClient()! }
```

Comment:
I don't get why this line is needed if the result is unused... side effects?
Maybe the ending `!` can be removed at least...

Reply:
removed